### DISCUSSION TITLE
Developing for NRF52811

### DISCUSSION THREAD

**Discussion by j45p41** [h2zero/NimBLE-Arduino#583] [Q&A]
Hi,

thanks again for a great library. I am developing using the nrf52840_DK and would like to use a nrf52811 SoC in my designs. 
Would I only need to change the variant file? can I just address those pins that are available on the nrf5211? if so where could I find the setting?

Thanks in advance!

**Comment by h2zero** [comment]
You would need to add a few files and some config stuff. I believe someone had done some work on this for the 52810, I will have a look when I have some time.
